<launch>

 <!-- Taken from https://wiki.ros.org/depth_image_proc#Execution -->
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />
  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <remap from="camera_info" to="/d435/camera/depth/camera_info"/>
    <remap from="image_rect" to="/d435/camera/depth/image_rect_raw"/>
    <remap from="points" to="/d435/camera/depth/points"/>
    <param name="queue_size" type="int" value="500"/>
  </node>
</launch>